game/src/main/resources/assets/sol_client/lang/fi_FI.lang

```diff
@@ -0,0 +1,442 @@
+sol_client.mod.zoom.option.zoomOutKey=Ulospäin Zoomaamisen näppäin
+sol_client.mod.zoom.zoom_in=Zoomaa sisään
+sol_client.mod.zoom.option.zoomInKey=Sisäänpäin Zoomaamisen näppäin
+sol_client.mod.zoom.option.cinematic=Elokuvakamera
+sol_client.mod.zoom.option.reduceSensitivity=Vähennä herkkyyttä
+sol_client.mod.zoom.option.scrolling=Vieritä Zoomataksesi
+sol_client.mod.zoom.option.smooth=Zoomaus Animaatio
+sol_client.mod.zoom.option.factor=Zoomaustekijä
+
+sol_client.mod.replay.name=Replay
+sol_client.mod.replay.description=Kuvaa ja toista pelaamisesi.
+
+sol_client.mod.replay.option.enableNotifications=Salli ilmoitukset
+sol_client.mod.replay.option.recordSingleplayer=Tallena yksinpeliä
+sol_client.mod.replay.option.recordServer=Tallena palvelinta
+sol_client.mod.replay.option.recordingIndicator=Tallennusilmaisimen
+sol_client.mod.replay.option.recordingIndicatorColour=Tallennusilmaisimen Väri
+sol_client.mod.replay.option.recordingIndicatorScale=Tallennusilmaisimen Koko
+sol_client.mod.replay.option.recordingIndicatorTextColour=Tallennusilmaisimen tekstin väri
+sol_client.mod.replay.option.recordingIndicatorTextShadow=Tallennusilmaisimen tekstin varjo
+sol_client.mod.replay.option.automaticRecording=Automaattinen tallennus
+sol_client.mod.replay.option.renameDialog=Uudelleen nimeä Dialogi
+sol_client.mod.replay.option.showChat=Näytä chat
+
+sol_client.mod.replay.option.camera=Kamera
+sol_client.mod.replay.option.camera.classic=Klassinen
+sol_client.mod.replay.option.camera.vanilla_ish=Vanilla-ish
+
+sol_client.mod.replay.option.showPathPreview=Näytä Polun Esikatselu
+
+sol_client.mod.replay.option.defaultInterpolator=Oletusinterpolaattori
+sol_client.mod.replay.interpolator.catmull=Catmull-Rom Spline
+sol_client.mod.replay.interpolator.cubic=Cubic Spline
+sol_client.mod.replay.interpolator.linear=Lineaarinen
+
+sol_client.mod.replay.option.showServerIPs=Näytä palvelimien IP-osoitteet
+
+sol_client.mod.quickplay.name=Quick Play
+sol_client.mod.quickplay.description=Liity nopeasti pelien jonoihin näppäimistön avulla.
+
+sol_client.mod.quickplay.key=Quick Play
+sol_client.mod.quickplay.option.menuKey=Näppäin
+
+sol_client.mod.hypixel_util.name=Hypixel Lisäykset
+sol_client.mod.hypixel_util.description=Useita parannuksia Hypixelin käyttökokemukseen.
+
+sol_client.mod.hypixel_util.option.visitHousingCommand=/visithousing
+sol_client.mod.hypixel_util.option.lobbySoundsVolume=Aulan äänenvoimakkuus
+sol_client.mod.hypixel_util.option.housingMusicVolume=Housing Musiikin Äänenvoimakkuus
+sol_client.mod.hypixel_util.option.popupEvents=Popup Tapahtumat
+sol_client.mod.hypixel_util.option.autogg=Auto GG
+sol_client.mod.hypixel_util.option.autoggMessage=GG viesti
+sol_client.mod.hypixel_util.option.hidegg=Piilota GG
+sol_client.mod.hypixel_util.option.autogl=Auto GL
+sol_client.mod.hypixel_util.option.autoglMessage=GL Viesti
+sol_client.mod.hypixel_util.option.hidegl=Piilota GL
+sol_client.mod.hypixel_util.option.levelhead=Level Head
```

Comment:
Maybe you could translate level head and auto.

Reply:
I think translating Level Head would lose the meaning. Auto GG and Auto GL too.